data/json/items/resources/tailoring.json

```diff
@@ -300,7 +313,8 @@
     "name": { "str": "Nomex sheet" },
     "description": "A sheet of Nomex synthetic fabric, suitable for making heat-resistant clothing.",
     "weight": "5 g",
-    "volume": "300 ml",
+    "volume": "5 ml",
+    "//": "Density of 0.72-1.1 grams per mL, using 0.91",
```

Comment:
Using 0.72 rather than the 0.91 average increases the volume by about 26%, far more than the 5% added volume you suggested for the far more rigid kevlar panels. You might want to increase that or reduce this?